### PR TITLE
Convert Dockerfile into a multistage one.

### PR DESCRIPTION
**Pull request by stephenwithav**
Removes need for builder pattern.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
+FROM golang:1.8
+RUN go get -u github.com/vimukthi-git/beanstalkg
+WORKDIR /src/github.com/vimukthi-git/beanstalkg
+COPY . /src/github.com/vimukthi-git/beanstalkg
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /beanstalkg .
+
 FROM scratch
```
Comment (vimukthi-git) [inline Dockerfile]: Shall we start from `https://hub.docker.com/r/centurylink/ca-certs/` image and remove the `ca-certificates.crt` file from the repo and the DockerFile?

Comment (stephenwithav) [inline Dockerfile]: Yes indeed!  Nice find!

Comment (stephenwithav) [inline Dockerfile]: Done.

**Comment by vimukthi-git** [comment]
@stephenwithav you can merge.

**Comment by stephenwithav** [comment]
Done.  Thank you, sir.

**Pull request merged**